server/server.js

```diff
@@ -0,0 +1,30 @@
+// Server
+const express = require('express')
+const app = express()
+const port = 5000
+const path = require('path');
+
+
+app.listen(port, () => {
+  console.log(`Example app listening on port ${port}`)
+})
+
+// Pages
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname, '../client/index.html'))
+})
+
+
+app.get('/coffee', (req, res) => {
+  res.sendFile(path.join(__dirname, '../client/coffee.html'))
```

Comment:
This code lines seems to be directing the server to displaying an html page stored on the client side. 
On the contrary, I would expect the client side *fetching* data from the server side (when server is launched).